reference/7.1/Microsoft.PowerShell.Core/About/about_Parameter_Sets.md

```diff
@@ -35,6 +35,8 @@ The following requirements apply to all parameter sets.
 - If no parameter set is specified for a parameter, the parameter belongs to
   all parameter sets.
 
+- Function with parameter sets specified must be called with parameters from a single parameter set.
```

Comment:
I don't think this actually fits. here. While this is true. This section is specifying the requirements for defining parameter sets, not calling them.

I'm not very sure that this needs to be in the article because as it says above, parameter sets **expose** different parameters to the user. Meaning that the other parameters won't work together with them. 

I'm open to discussion on it and might see some value by adding something along the lines of:

`Parameters unique to a parameter set can't be used with parameters unique to a different parameters set.`

Like I said, open to discussion and your thoughts on it too.

Reply:
I think your wording is better. I didn't mean to specify how it should be called. I thought it'd be valuable for the reader to understand what parameter sets do.